### PR TITLE
Fix parsing Ubuntu Impish (and higher) deb pkgs

### LINKED ISSUE #59: Ubuntu Impish (and higher) packages: no entry control.tar.xz in archive
When I am trying to create a repo of Tarantool packages for Ubuntu Impish, I am getting the following error:
```
Adding: 'pool/impish/main/t/tarantool/tarantool-common_2.10.0.g0a5ce0b9c-1_all.deb'
no entry control.tar.xz in archive
xz: (stdin): File format not recognized
tar: Child returned status 1
tar: Error is not recoverable: exiting now
Can't parse 'pool/impish/main/t/tarantool/tarantool-common_2.10.0.g0a5ce0b9c-1_all.deb':
Command 'ar -p ./.rws_32721p7q/tmpfrzfaeyn/package.deb control.tar.xz |tar -xJf - --to-stdout ./control' returned non-zero exit status 2.
```

It looks like control TAR archive for Ubuntu Impish (and higher) has the `zst` compression instead of `xz`.

See for details: https://balintreczey.hu/blog/hello-zstd-compressed-debs-in-ubuntu

### PR DESCRIPTION
**Pull request by ylobankov**
It looks like control TAR archive for Ubuntu Impish (and higher) 
has the `zst` compression instead of `xz`. This patch fixes the issue.

Fixes #59

### CODE REVIEW

**Review by Totktonada — APPROVED** [review]
Visually it looks ok. Did you test it manually? Please, share results.

**Comment by Totktonada** [comment]
@knazarov said in a private conversation that he does not mind the change.

**Comment by ylobankov** [comment]
> Visually it looks ok. Did you test it manually? Please, share results.

Yep, it was tested on the openssl package from Ubuntu Impish repo.

**Review by LeonidVas — COMMENTED** [review]
Hi! Thank you for the patch.
Several nits:
* Commit title - I suggest to write it according to https://github.com/tarantool/tarantool/wiki/Code-review-procedure#commit-message (`deb: fix...`). It is just convenient to have the same commit format in all the repositories.
* You can add a link to https://balintreczey.hu/blog/hello-zstd-compressed-debs-in-ubuntu/  Up to you.
* Add tests for the `parse_deb` method.

File: test/test_debrepo.py [hunk 1]
```diff
@@ -53,12 +53,29 @@ def test_versions_parsing_on_ubuntu_groovy(self):
...
+        pkg_names = [
+            'openssl_1.1.1f-1ubuntu2_amd64.deb',  # control.tar.xz
+            'openssl_1.1.1l-1ubuntu1_amd64.deb'  # control.tar.zst
+        ]
+        pkg_obj = debrepo.Package()
+        for pkg_name in pkg_names:
+            try:
+                pkg_obj.parse_deb(os.path.join(self.test_dir, f"resources/{pkg_name}"))
+            except FileNotFoundError:
+                self.fail('parse_deb() raised FileNotFoundError unexpectedly!')
```
Comment (Totktonada) [inline test/test_debrepo.py:72]: How about checking return values?

Comment (ylobankov) [reply test/test_debrepo.py:72]: Checking return values is not in the scope of this PR. I just added one more variant of extracting the `control.tar` archive. I didn't touch `self.parse_string()` method.

File: test/test_debrepo.py [hunk 2]
```diff
@@ -53,12 +53,29 @@ def test_versions_parsing_on_ubuntu_groovy(self):
...
 
         for test_file_name in test_file_names:
-            test_path = os.path.join(cur_dir, test_file_name)
+            test_path = os.path.join(self.test_dir, test_file_name)
             with open(test_path, 'r') as test_file:
                 for package_name in test_file:
                     self.assertIsNotNone(debrepo.split_control_file_path(package_name, 'binary'),
                                          "Can't parse packagename: %s" % package_name)
 
+    def test_control_tar_archive(self):
```
Comment (LeonidVas) [inline test/test_debrepo.py:62]: It looks like this test has nothing to do with the class `TestVersionParsing` I think a new class should be created `TestIndexUnit` and within that class `test_control_tar_archive` would be relevant.

File: test/test_debrepo.py (hunk 1 above)
Comment (LeonidVas) [reply test/test_debrepo.py:72]: In fact, it looks like a bit of work from above, thanks to which you can find out that the method not only does not throw exceptions, but also works.

**Review by LeonidVas — CHANGES_REQUESTED** [review]

File: test/test_debrepo.py (hunk 2 above)
Comment (ylobankov) [reply test/test_debrepo.py:65]: You are absolutely right. Thanks, fixed.

File: test/test_debrepo.py (hunk 1 above)
Comment (ylobankov) [reply test/test_debrepo.py]: Ok guys, fixed.

**Review by LeonidVas — APPROVED** [review]
LGTM

**Pull request merged**